runtime/src/main/java/org/corfudb/runtime/view/stream/StreamAddressSpace.java

```diff
@@ -47,11 +47,12 @@ private StreamAddressSpace(long trimMark, Roaring64NavigableMap bitmap) {
     }
 
     public StreamAddressSpace() {
-        this(Address.NON_ADDRESS, Roaring64NavigableMap.bitmapOf());
+        this(Address.NON_ADDRESS, new Roaring64NavigableMap(false, false));
```

Comment:
This is also used by the streaming layer. I have added a separate constructor that takes a boolean to determine if caching should be used by internal `StreamAddressSpace` data structures. I have also added a field to keep track of this parameter so that it can be passed into child `StreamAddressSpace` objects created by a particular instance.